docs/change-log/index.md

```diff
@@ -883,6 +883,7 @@ with a drill-down to view the steps and their details. [Tech Preview]
 |ML-4956|A function created by SDK is initially in the "initialized" state in the UI and needs to be deployed before running it. | In **Edit**, press **Deploy** | v1.5.1 |
 |ML-5079|Cannot update git remote with `project.create_remote()`| NA | v1.5.1 |
 |ML-5204|The **Projects>Settings** does not validate label names. Errors are generated from the back end. |Use [Kubernetes limitations](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set).  | v1.6.0 |
+|ML-5573|When using the validator flag to determine how to validate the data for the specific feature, the validate check runs twice on the same dataframe and therefore prints the same message twice.|NA| v1.6.0 |
```

Comment:
The default value of feature-set ingest() infer_options is "all" (which includes Preview) and as a result, during ingest, preview is done as well. Due to this, if a validator was configured for a feature, each violation will cause two messages to be printed.